pytket/phir/phirgen.py

```diff
@@ -22,7 +22,8 @@
     tk.OpType.CZ:       "CZ",
     tk.OpType.H:        "H",
     tk.OpType.PhasedX:  "R1XY",
-    tk.OpType.Reset:    "Reset",  # TODO(kartik): confirm with Ciaran
+    tk.OpType.Reset:    "Reset",  # TODO(kartik): confirm with Ciaran/spec
+                                  # https://github.com/CQCL/phir/blob/main/phir_spec_qasm.md
```

Comment:
`Reset` is currently not supported by PHIR spec and is rejected by my changes at https://github.com/CQCL/phir/pull/14. Will need Ciaran's input on the best way to handle resets.